Add setup.py for mac executable. Update readme.md to include informat… (#73)
ThePhilgrim/minesweeper: README.md

- There is a big and small version of each image. The big images are used to produce the small images.
- If you change the sizes of the small images, change `minesweeper.py` to use the new size.
- When resizing a big image to make a small image, make sure that the small image has transparent background.

> - Keep `image_sources.txt` up to date.

Reviewer: ```suggestion
- Keep `images/sources.txt` up to date.
```

bad merge, sorry

## Notes about creating executable file on Mac OS

To reproduce the installer, please run the following commands: (for reference: https://www.youtube.com/watch?v=DVOoHL2Bp_o&t=461s&ab_channel=SamanthaCruz, http://www.marinamele.com/from-a-python-script-to-a-portable-mac-application-with-py2app)
- The executable is created with py2app. Run `pip3 install -U py2app` in terminal to install.

> - Make sure that you are located in the minesweeper directory. In your terminal, run `python3 setup.py py2app

Reviewer: ```suggestion
- Make sure that you are located in the minesweeper directory. In your terminal, run `python3 setup.py py2app`
```

- If you change the sizes of the small images, change `minesweeper.py` to use the new size.
- When resizing a big image to make a small image, make sure that the small image has transparent background.
- Keep `image_sources.txt` up to date.

## Notes about creating executable file on Mac OS

To reproduce the installer, please run the following commands: (for reference: https://www.youtube.com/watch?v=DVOoHL2Bp_o&t=461s&ab_channel=SamanthaCruz, http://www.marinamele.com/from-a-python-script-to-a-portable-mac-application-with-py2app)
- The executable is created with py2app. Run 'pip3 install -U py2app' in terminal to install.

> - Make sure that you are located in the minesweeper directory. In your terminal, run 'python setup.py py2app'

Reviewer: ```suggestion
- Make sure that you are located in the minesweeper directory. In your terminal, run `python setup.py py2app`
```

This makes the code appear with a different font in the README.
